plugins/value-container-assignment/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+description = "Kotlin Value Container Assignment Compiler Plugin"
+
+plugins {
+    kotlin("jvm")
+    id("jps-compatible")
+}
+
+dependencies {
+    embedded(project(":kotlin-value-container-assignment-compiler-plugin.common"))
+    embedded(project(":kotlin-value-container-assignment-compiler-plugin.k1"))
+    embedded(project(":kotlin-value-container-assignment-compiler-plugin.k2"))
+    embedded(project(":kotlin-value-container-assignment-compiler-plugin.cli"))
+
+    testApi(project(":compiler:backend"))
+    testApi(project(":compiler:cli"))
+    testApi(project(":kotlin-value-container-assignment-compiler-plugin.cli"))
+    testCompileOnly(project(":kotlin-compiler"))
+    testImplementation(project(":kotlin-scripting-jvm-host-unshaded"))
+
+    testApiJUnit5(vintageEngine = true)
+
+    testApi(projectTests(":compiler:tests-common-new"))
+    testApi(projectTests(":compiler:test-infrastructure"))
+    testApi(projectTests(":compiler:test-infrastructure-utils"))
+
+    testImplementation(projectTests(":compiler:tests-common"))
```

Comment:
Those two dependencies are also look redundant

Reply:
Removed

Reply:
`":compiler:tests-common"` and `junit:junit` should be removed

Reply:
Also please remove dependencies on `kotlin-reflect` and `kotlin-reflect-api`